cfme/cloud/provider.py

```diff
@@ -75,8 +76,7 @@
         ('google_project_text', Input("project")),
         ('azure_tenant_id', Input("azure_tenant_id")),
         ('azure_subscription_id', Input("subscription")),
-        ('amazon_region_select', {version.LOWEST: Select("select#provider_region"),
-            "5.5": AngularSelect("provider_region")}),
+        ('amazon_region_select', AngularSelect("ems_region")),
```

Comment:
As this is called ```ems_region``` implying that it doesn't matter which provider uses it, we should probably rename this one too. Those strings are easily greppable in the codebase and I'd have no worry merging those changes.